compiler/trees.nim

```diff
@@ -124,8 +124,16 @@ proc isRange*(n: PNode): bool {.inline.} =
       result = true
 
 proc whichPragma*(n: PNode): TSpecialWord =
+  # echo "whichPragma"
```

Comment:
don't keep temporary debugging statements (or mark the PR as WIP and add something to make it clear to reviewer to ignore those as well as for you to remember to remove in final version, eg:
# echo "key"
=>
# echo "key" # PRTEMP